specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-03-01-preview/Capabilities.json

```diff
@@ -0,0 +1,538 @@
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "StorageTierCapability": {
+      "type": "object",
+      "description": "Represents capability of a storage tier",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name to represent Storage tier capability",
+          "readOnly": true
+        },
+        "iops": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Supported IOPS for this storage tier",
+          "readOnly": true
+        }
+      }
+    },
+    "StorageMbCapability": {
+      "type": "object",
+      "properties": {
+        "supportedIops": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Supported IOPS",
+          "readOnly": true
+        },
+        "storageSizeMb": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Storage size in MB",
+          "readOnly": true
+        },
+        "defaultIopsTier": {
+          "type": "string",
+          "description": "Default tier for IOPS",
+          "readOnly": true
+        },
+        "supportedIopsTiers": {
+          "type": "array",
+          "description": "List of available options to upgrade the storage performance",
+          "items": {
+            "$ref": "#/definitions/StorageTierCapability"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ],
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../types/common-types.json#/definitions/CapabilityBase"
+        }
+      ],
+      "description": "storage size in MB capability"
+    },
+    "ServerSku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Sku name",
+          "readOnly": true
+        },
+        "vCores": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Supported vCores",
+          "readOnly": true
+        },
+        "supportedIops": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Supported IOPS",
+          "readOnly": true
+        },
+        "supportedMemoryPerVcoreMb": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Supported memory per vCore in MB",
+          "readOnly": true
+        },
+        "supportedZones": {
+          "type": "array",
+          "description": "List of supported Availability Zones",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true
+        },
+        "supportedHaMode": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Supported high availability mode",
```

Comment:
>  "description": "Supported high availability mode", [](http://example.com/codeflow?start=8&length=52)

What are the valid values for this property that someone should understand? Should this be an enumeration?

Reply:
Added an enum

Reply:
I don't see a change for the supportedHaMode property which is what I was trying to comment on.

Reply:
Pushed the commit